projects/template.md

```diff
@@ -132,6 +132,28 @@ submitted.
     * If selected, please attach a paragraph detailing the information which will be presented and any steps which will be taken to prevent this from potentially misleading the public.
   2. [ ] Project results will not present information which might lead to PKT price speculation
 
+## Use of Resources
+Each milestone report will be accompanied by:
+
+* [ ] Itemization of project expenditures, including description, date and price (in national currency)
+  * [ ] Justification of this itemization in an audit certificate
+* [ ] Dates when PKT was converted to national currency
+  * [ ] Justification of this conversion in an audit certificate
+* [ ] An audit certificate from a qualified financial auditor
+  * *If yes, please provide name of the auditor*: ___________
+
+> If there is any other information which you can agree to provide that will help the network
+steward validate your use of resources, please add it here.
+
+### Partial milestones
```

Comment:
Lots of the indentation is messed up, perhaps this is an opportunity to fix them all?
See: https://github.com/pkt-cash/ns-projects/blob/073fe3662e67a8c0dfbb0abd97cd2a070a19e752/projects/template.md#disclosure

Reply:
Can you make another PR to do that so we're not blocking acceptance of opening a new round on it ?

Reply:
Feel free to make changes in another future PR, and accept this as is.